src/common/hooks/useRestoreToken.tsx

```diff
@@ -44,34 +44,66 @@ export const useRestoreToken = (
   const [errorMessage, setErrorMessage] = useState<string>();
   const [state, setState] = useState<ContractFunctionState>("UNINITIALIZED");
 
+  const sendToTitleEscrow = async (previousBeneficiary: string, previousHolder: string): Promise<void> => {
+    if (!tokenId) throw new Error("Ownership data is not provided");
+    if (!contractInstance?.address) throw new Error("Token Registry Instance should have address");
+    setState("PENDING_CONFIRMATION");
+
+    const sendToNewEscrowReceipt = await contractInstance?.sendToNewTitleEscrow(
+      previousBeneficiary,
+      previousHolder,
+      tokenId
+    );
+
+    const sendToNewEscrowTx = await sendToNewEscrowReceipt.wait();
```

Comment:
The transaction is pending confirmation before this line and confirmed after this line. The `.wait()` is waiting for the transaction to be mined.